torch/_decomp/decompositions.py

```diff
@@ -288,6 +288,28 @@ def _prelu_kernel_backward(
     return (input_grad, weight_grad)
 
 
+@register_decomposition(aten.rrelu_with_noise)
+@pw_cast_for_opmath
+def rrelu_with_noise(
+    self: Tensor,
+    noise: Tensor,
+    lower: float,
+    upper: float,
+    training: bool=False,
+    generator: Optional[torch.Generator]=None,
+) -> Tensor:
+    assert generator is None
+    if training:
+        not_positive = self <= 0
+        r = aten.uniform(self, lower, upper)
```

Comment:
note, this will have different rng semantics compared to eager pytorch, since it's using the rng for each element in self as opposed to just the not positive ones. 

I'm not sure of the precedent for that as far as general decompositions go, but I think it's fine - see [Caution - These decomps do not have same accuracy as that of eager. However, # we can't just disable them with a config flag like fallback_random](https://github.com/pytorch/pytorch/blob/main/torch/_decomp/decompositions_for_rng.py#L221)

Maybe it's worth adding a test checking the behavior we expect from this decomp in `test_decomp`?